doc/version_history.rst

```diff
@@ -6,6 +6,24 @@
 Version History
 ###############
 
+v2.8.6
+------
+
+Changes:
+
+* In `QueueModel._log_message_callback` use `self.log.debug` instead of `print`.
```

Comment:
The verbose option prints to stdout on purpose, to avoid duplicating messages which are already being sent to the log via the SAL logging system.

Reply:
You are right. The log messages were all coming out at the end, where they did no good. Thus all the print statements and verbose option.